src/include/distributed/commands.h

```diff
@@ -747,6 +747,8 @@ extern void ExecuteAndLogUtilityCommand(const char *commandString);
 extern void ExecuteForeignKeyCreateCommandList(List *ddlCommandList,
 											   bool skip_validation);
 
+extern void CreateNullShardKeyDistTable(Oid relationId, char *colocateWithTableName);
```

Comment:
I know metadata_utility.h doesn't like the right place but I feel like we should define this right after `extern CreateDistributedTable` in the same file.